src/components/UI/SearchDialogBox/SearchDialogBox.tsx

```diff
@@ -68,6 +86,14 @@ export const SearchDialogBox = (props: SearchDialogBoxProps) => {
 
   const changeValue = (event: any, value: any) => {
     setSelectedOptions(value);
+    if (multiple) {
```

Comment:
Looks like we are duplicating the code here (line 66)

Reply:
the useEffect is for when options are loaded for the first time and the function is for when option is selected